resources/assets/js/misc/datatable/delete.js

```diff
@@ -0,0 +1,34 @@
+exports.deleteItemFromDataTable = (dataTable) => {
+    $(document).on('click', '.js-delete-resource', function (e) {
+        e.preventDefault();
+        let url = $(this).data('url');
+
+        swal({
```

Comment:
You could use async/await here. That way you could get rid of the callback. As in this example https://www.dropbox.com/s/prc6xp3qgctszm4/Screenshot%202019-02-10%2008.39.30.png?dl=0